datahub-web-react/src/app/search/EntityGroupSearchResults.tsx

```diff
@@ -0,0 +1,90 @@
+import { ArrowRightOutlined } from '@ant-design/icons';
+import { Button, Card, Divider, List, Space, Typography } from 'antd';
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+import { SearchCfg } from '../../conf';
+import { useGetSearchResultsQuery } from '../../graphql/search.generated';
+import { EntityType } from '../../types.generated';
+import { IconStyleType } from '../entity/Entity';
+import { useEntityRegistry } from '../useEntityRegistry';
+import { navigateToSearchUrl } from './utils/navigateToSearchUrl';
+
+const styles = {
+    header: { marginBottom: 20 },
+    resultHeaderCardBody: { padding: '16px 24px' },
+    resultHeaderCard: { right: '52px', top: '-40px', position: 'absolute' },
+    resultList: { width: '100%', borderColor: '#f0f0f0', marginTop: '12px', padding: '16px 32px' },
+    seeAllButton: { fontSize: 18 },
+    resultsContainer: { width: '100%', padding: '40px 132px' },
+};
+
+interface Props {
+    type: EntityType;
+    query: string;
+}
+
+export const EntityGroupSearchResults = ({ type, query }: Props) => {
+    const history = useHistory();
+    const entityRegistry = useEntityRegistry();
+    const { data } = useGetSearchResultsQuery({
+        variables: {
+            input: {
+                type,
+                query,
+                start: 0,
+                count: SearchCfg.RESULTS_PER_PAGE,
+                filters: null,
+            },
+        },
+    });
+
+    const results = data?.search?.entities || [];
+
+    return (
+        <Space direction="vertical" style={styles.resultsContainer}>
+            <List
+                header={
+                    <span style={styles.header}>
+                        <Typography.Title level={3}>{entityRegistry.getCollectionName(type)}</Typography.Title>
+                        <Card bodyStyle={styles.resultHeaderCardBody} style={styles.resultHeaderCard as any}>
+                            {entityRegistry.getIcon(type, 36, IconStyleType.ACCENT)}
+                        </Card>
+                    </span>
+                }
+                footer={
+                    data?.search &&
+                    data?.search?.total > 0 && (
```

Comment:
should we show a `show all` button in the case that there's only one result?

Reply:
or should this be total > SearchCfg.RESULTS_PER_PAGE?

Reply:
hm....... see all also serves the purpose of flipping tabs for you so you can apply filters etc.. i think see all is still okay...